kolibri/core/package.json

```
@@ -21,7 +21,7 @@
     "js-cookie": "^3.0.5",
     "knuth-shuffle-seeded": "^1.0.6",
     "kolibri-constants": "0.2.6",
-    "kolibri-design-system": "4.3.2",
+    "kolibri-design-system": "5.0.0-rc1 ",
```

Seems like there is an extra space, I dont think this could cause a problem, but just in case.
```suggestion
    "kolibri-design-system": "5.0.0-rc1",
```

---

Thanks, yes better to fix. Done